CHANGELOG.md

```diff
@@ -10,6 +10,47 @@
 
 - iOS only: Logins store has a new (static) `numOpenConnections` function, which can be used to detect leaks. ([#1070](https://github.com/mozilla/application-services/pull/1070))
 
+## Megazords
+
+### Breaking Changes
+
+- Megazord initialization is now separate from the initialization of the http client. ([#1066](https://github.com/mozilla/application-services/issues/1066))
+    - This is done to allow application that wish to defer initializing JNA
+      (which can be costly) until later. Now, you should use the `RustHttpConfig.setClient()` function from `org.mozilla.appservices:httpconfig`:
```

Comment:
Would we expect android-components to provide a wrapper around this, like it currently does for e.g. rustlog?

Reply:
Hmm, it's just a single function, rustlog has more complex initialization. Eventually we might want to consider it, but in the short term I don't see the benefit other than consolidating initialization functions.

I'd almost like to see us move rustlog into our repo, and consolidate initialization functions that way. Not sure how easy that is though, it seemed hard at the time (I got weird errors, but these might have been unrelated in retrospect -- I saw similar issues when importing concept-fetch initially, but they were just gradle config problems).

This would have a secondary benefit of letting us use a-c log in our components, probably.

Reply:
One wrinkle here may be https://github.com/mozilla-mobile/fenix/pull/1942 in which @grigoryk removed any direct dependency on a-s, in favour of pulling everything in via a-c. Perhaps that becomes less of an issue with the runtime checks you're adding here about version compatibility.

Reply:
Well, I think they'll need a direct dependency on a-s for the megazord anyway.